api/src/shop/test/stripe_customer_test

```diff
@@ -0,0 +1,64 @@
+from logging import getLogger
```

Comment:
Seems like the test is missing a file ending (.py)?

Reply:
Yeah I forgot it